packages/models/src/models/Users.ts

```diff
@@ -3410,4 +3410,16 @@ export class UsersRaw extends BaseRaw<IUser, DefaultFields<IUser>> implements IU
 			},
 		);
 	}
+
+	countActiveUsersByNotDMRoomId(rid: string) {
```

Comment:
Can we have a better name for this method? It's not clear on what it's filtering... 🤔